src/main/java/db/FruitShopDaoCsvImpl.java

```diff
@@ -0,0 +1,22 @@
+package db;
+
+import java.util.Collections;
+import java.util.Map;
+import storage.Storage;
+
+public class FruitShopDaoCsvImpl implements FruitShopDao {
+    @Override
+    public void add(String fruit, Integer quantity) {
+        Storage.fruitQuantities.put(fruit, quantity);
+    }
```

Comment:
Better use 'put' for a method with this logic. 'add' is suitable for adding (storageQuantity + newQuantity)